schema_utils.py

```diff
@@ -236,8 +236,10 @@ def has_length_at_most(obj, max_value):
 
     @staticmethod
     def is_nonempty(obj):
-        """Returns True iff the given object (a string) is nonempty."""
-        return bool(obj)
+        """Always returns True so that an input field can be made required
```

Comment:
This function checks for the default value and for the new implementation we need the function to return true even for strings of zero length right?
 
The test fails here in line 170 of  schema_utils_test.py if we don't provide a default. 
assert set(ui_config.keys()) <= set(reference_dict.keys())

The is_nonempty method returns a null value and this is the assertion error we get.
assert set(ui_config.keys()) <= set(reference_dict.keys())
AssertionError